contracts/ColonyNetwork.sol

```diff
@@ -246,6 +247,9 @@ contract ColonyNetwork is ColonyNetworkStorage {
   calledByColony
   skillExists(_skillId)
   {
+    /* if (_amount ==0 ){
+      return;
+    } */
```

Comment:
Leftover from debugging perhaps?

Reply:
Less debugging, more me exploring the concept of 0-reputation change entries not being added to the log for gas reasons. But 

1. I realised it was better to not even make the call to this contract - check if it's zero locally, and then make the call if it's not.
2. It messes up a lot of tests.

The solidity change is probably good-for-collaborators, but the number of tests that would require changing gives me pause.

Reply:
I don't see the zero checks being done. Shall we log an issue? Collaborators should be fine with test cases I mean `appendReputationUpdateLog` is only called from `finalizeTask` and `bootstrapColony` right? Where is the complexity there?

Reply:
The zero checks are not currently done anywhere, no. The tests it breaks are all the tests that look at the reputationUpdateLog as part of the test passing, or the length of that log.